runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -1094,16 +1094,14 @@ registerJdkInternalMiscUnsafeNativesJava14(JNIEnv *env, jclass clazz) {
 void JNICALL
 Java_jdk_internal_misc_Unsafe_registerNatives(JNIEnv *env, jclass clazz)
 {
-	J9VMThread *currentThread = (J9VMThread*)env;
-
 	Java_sun_misc_Unsafe_registerNatives(env, clazz);
 	registerJdkInternalMiscUnsafeNativesCommon(env, clazz);
-	if (J2SE_VERSION(currentThread->javaVM) >= J2SE_V11) {
+	if (JAVA_SPEC_VERSION >= 10) {
+		if (JAVA_SPEC_VERSION >= 14) {
+			registerJdkInternalMiscUnsafeNativesJava14(env, clazz);
+		}
```

Comment:
Please don't reorder these calls. Also, there's no value in nesting the tests - both are compile-time constants.
```
	if (JAVA_SPEC_VERSION >= 10) {
		registerJdkInternalMiscUnsafeNativesJava10(env, clazz);
	}
	if (JAVA_SPEC_VERSION >= 14) {
		registerJdkInternalMiscUnsafeNativesJava14(env, clazz);
	}
```

Reply:
Updated.